packages/node_modules/pouchdb-wrappers/lib/index.js

```diff
@@ -105,15 +105,14 @@ wrapperBuilders.destroy = function (db, destroy, handlers) {
 };
 
 wrapperBuilders.put = function (db, put, handlers) {
-  return function (doc, docId, docRev, options, callback) {
+  return function (/*doc, docId, docRev, options, callback*/) {
```

Comment:
Should the comment be removed?

Reply:
Not really, it helps understanding the function body. As it uses 'arguments' to actually access the parameters, though, my editor's linter freaked out. This was my compromise.